mods/content/polaris/datum/languages/human.dm

```diff
@@ -112,11 +112,111 @@
 		"wa", "wa", "wa", "wa", "wa", "wa", "wa", "wai", "wan", "wang", "wei", "wen", "weng", "wo", "wu",
 		"xi", "xia", "xian", "xiang", "xiao", "xie", "xin", "xing", "xiong", "xiu", "xu", "xuan", "xue", "xun",
 		"ya", "yan", "yang", "yao", "ye", "yeg", "yeg", "yeg", "yey", "yey", "yey", "yi", "yin", "ying", "yong", "you", "yu", "yuan", "yue", "yun",
-		"za", "zai", "zan", "zang", "zao", "ze", "zei", "zen", "zeng", "zi", "zong", "zou", "zu", "zuan", "zui", "zun", "zuo", 
+		"za", "zai", "zan", "zang", "zao", "ze", "zei", "zen", "zeng", "zi", "zong", "zou", "zu", "zuan", "zui", "zun", "zuo",
 		"zha", "zhai", "zhan", "zhang", "zhao", "zhe", "zhei", "zhen", "zheng", "zhi",
 		"zhong", "zhou", "zhu", "zhua", "zhuai", "zhuan", "zhuang", "zhui", "zhun", "zhuo"
 	)
 
+/decl/language/human/tradeband
+	name = "Tradeband"
+	uid = "language_tradeband"
+	desc = "Spoken by the humans of the upper-class Sagittarius Heights, Tradeband was designed to be pleasing to both humans and their Skrellian trading partners."
+	speech_verb = "enunciates"
+	colour = ""
+	key = "2"
+	partial_understanding = list("Galactic Common" = 40, "Solar Common" = 20, "Terminus" = 40, "Gutter" = 10, "Sivian Creole" = 10)
+	//, LANGUAGE_SKRELLIAN = 40, LANGUAGE_SKRELLIANFAR = 15
```

Comment:
Galcom should also have the Skrellian bits, I take it these values are from old code?

Is there an issue with including Skrellian being "not found" due to it being in another modpack? We may have to define our own version of Skrellspeak in the polaris modpack if that's the case, but that's an issue for a Skrell PR.

Reply:
...I kind of forgot they were in-game already, I'll add them in